app-frontend/src/app/pages/projects/edit/aoi-approve/aoi-approve.html

```diff
@@ -0,0 +1,32 @@
+<div class="sidebar-project">
+  <div class="sidebar-header">
+    <a class="btn sidebar-header-nav-btn" ui-sref="projects.edit">
+      <i class="icon-arrow-left"></i>
+    </a>
+    <h5 class="sidebar-title">AOI Scene Approval</h5>
```

Comment:
This is a good question. I think writing out 'Area of Interest' should be done where possible for clarity. The mockups use 'AOI' and 'Area of Interest' in different places, but I'm not sure we've thought much about where abbreviations for this are and are not appropriate. Definitely something to think about. We do have some space limitations where we would like to prevent the text from going to multiple lines.